cfgov/preprocessed/css/nav-secondary.less

```diff
@@ -194,10 +194,14 @@
     }
 
     &_item__parent {
-        .h4();
         .webfont-medium();
-        margin-bottom: 0;
+
         padding-bottom: 0;
+        margin-bottom: 0;
+
+        .respond-to-min(@desktop-min, {
+          font-size: 18px;
```

Comment:
Not really. It was `h4()` then reset to medium instead of demi weight, which doubled up on the font-family. `h4()` should only be used it it's actually `h4()` styles, and not just sizing. Plus it's supposed to be 16px at small screens and h4 doesn't ever change size.